.github/workflows/try-release.yml

```diff
@@ -20,5 +20,5 @@ jobs:
       - name: Build .deb
         id: build
         env:
-          GO_VERSION: 1.19.5
```

Comment:
While we're here, let's make this a matrix just like it is in release.yml

Reply:
Can do